js/sign.js

```diff
@@ -0,0 +1,52 @@
+const errorMessageEmail = document.querySelector('.enter-email');
+const errorMessageEmail2 = document.querySelector('.wrong-email');
+const emailInput = document.querySelector('.email-input');
+
+emailInput.onblur = function (e) {
+  const regex = new RegExp('[a-z0-9]+@[a-z]+\.[a-z]{2,3}');
```

Comment:
먼저 상수이기때문에 대문자로 작성해주시면 좋을 것 같아요!
```suggestion
  const REGEX = new RegExp('[a-z0-9]+@[a-z]+\.[a-z]{2,3}');
```

Reply:
그리고 지금은 정규식이라는 의미만 있고 어떤 정규식인지는 변수명으로 알기가 어려운데요! 
`EMAIL_REGEX`와 같은 이름이 좋을 것 같아요!

```suggestion
  const EMAIL_REGEX = new RegExp('[a-z0-9]+@[a-z]+\.[a-z]{2,3}');
```

Reply:
요렇게 표현할 수도 있어요~

```suggestion
  const EMAIL_REGEX = /[a-z0-9]+@[a-z]+\.[a-z]{2,3}/
```